### ISSUE TITLE
Video stops updating - audio keeps playing

### ISSUE THREAD

**Issue by asampal** [jackun/VLCTube#49]
In this video, https://www.youtube.com/watch?v=CoJyONmcGbM, I keep having the video stop updating, but the audio keeps on going. This happened at different points in the video and it was not consistent between restarts of the video. I was playing at 2x speed.

Using Windows 8.1 64 bit, VLC 2.20

**Comment by Sonic0170** [comment]
i tested the video at 2x speed as you described 2 times in a row and could not recreate the issue.

**Comment by asampal** [comment]
I just tried this video again and it happened. I forgot to mention probably the most important detail here, when I reported this, I was using Firefox 37 beta 5, but it happens with beta 6 as well.

**Comment by jackun** [comment]
I don't think this is specifically related to this script. Just the way how VLC operates and network conditions etc. I usually get audio "desync" when VLC buffer underruns and have to seek to get audio back again.

**Comment by asampal** [comment]
Well, this has never happened to me before and the factors that have changed are VLC and Firefox. So maybe it's just some assumptions wrt buffering/caching that are off. 

@jackun, when you say that you get audio "desync", do you mean that the audio gets out of sync with the video or, like for me, that the video stops updating altogether?

**Comment by jackun** [comment]
It should be a known issue with VLC. Audio timestamps get out of sync (or something like that) and VLC silences the audio. I think.

**Comment by asampal** [comment]
@jackun, so you're saying that what you've seen is the audio stopping, not the video, right? Have you ever had the video stream stop, but the audio still plays (i.e. playback head is advancing and audio keeps playing)?

**Comment by Sonic0170** [comment]
ah yeah im using v36.0.1 stable, im also on windows 7 ultimate 64bit
I have started avoiding the nightly, aurora and beta releases because mozilla is changing many security features that are constantly breaking VLCTube and various other plugins, although i keep nightly installed and up to date for testing purposes.
in this case i assume its firefox beta that is causing this issue since this issue doesn't exist (for me at least) in the stable release yet.

**Comment by Sonic0170** [comment]
im also using vlc nightly v2.2.0 1/26/2015, haven't updated to newer versions yet and im wary of v3.0 since it seems to be really buggy so far
